Doc/whatsnew/3.10.rst

```diff
@@ -1223,6 +1223,12 @@ now call :func:`inspect.get_annotations` to retrieve annotations. This means
 also now un-stringize stringized annotations.
 (Contributed by Larry Hastings in :issue:`43817`.)
 
+itertools
+---------
+
+Add :func:`itertools.pairwise()`.
+(Contributed by Brett Cannon in :issue:`38200`.)
```

Comment:
Wasn't this Raymond Hettinger, not Brett Cannon?

Reply:
Yup https://github.com/python/cpython/pull/23549

Reply:
You are very correct, I duplicated the section right below but seems like I neglected to replace the name. Sorry Raymond!